### PR TITLE
Allow for comment deletion

### LINKED ISSUE #224: Delete comments

### PR DESCRIPTION
**Pull request by fdb-hiroshima**
Fix #224 

Current design is a bit terrible. I've tried things, only made it worse. If someone better than me at CSS could give it a try

This tries to conserve as much threads as possible, by re-linking comments to the parent of the one being deleted

### CODE REVIEW

**Comment by igalic** [comment]
we could do the reddit thing and leave a greyed out `deleted` box. that might be easiest.

**Comment by fdb-hiroshima** [comment]
My problem is not with deleted comment, I just delete them. It's with the delete button itself

File: templates/partials/comment.rs.html [hunk 1]
```diff
@@ -3,14 +3,14 @@
...
-@(ctx: BaseContext, comm: &Comment, author: User, in_reply_to: Option<&str>)
+@(ctx: BaseContext, comm: &Comment, author: User, in_reply_to: Option<&str>, blog: &str, slug: &str)
 
 <div class="comment u-comment h-cite" id="comment-@comm.id">
     <a class="author u-author h-card" href="@uri!(user::details: name = author.get_fqn(ctx.0))">
         @avatar(ctx.0, &author, Size::Small, true, ctx.1)
         <span class="display-name p-name">@author.name(ctx.0)</span>
         <small>@author.get_fqn(ctx.0)</small>
-    </a>
+	    </a>
```
Comment (igalic) [inline templates/partials/comment.rs.html]: why this indent?

File: plume-models/src/comments.rs [hunk 2]
```diff
@@ -1,4 +1,4 @@
-use activitypub::{activity::Create, link, object::Note};
+use activitypub::{activity::{Create, Delete}, link, object::{Note, Tombstone}};
```
Comment (igalic) [inline plume-models/src/comments.rs]: Tombstone… sounds cool

Comment (fdb-hiroshima) [reply plume-models/src/comments.rs]: It's what you need to live behind when deleting things (as opposed to undoing)

File: templates/partials/comment.rs.html (hunk 1 above)
Comment (fdb-hiroshima) [reply templates/partials/comment.rs.html]: There is a 100% chance my vim is still fucking up the indent in html templates, I have no idea why but in those rs.html it absolutely want to add tabs everywhere

**Comment by fdb-hiroshima** [comment]
No, that was all

**Pull request merged**